typescript/src/main.ts

```diff
@@ -45,7 +45,46 @@ function appendMetalTableRow(metal: metals.Metal, table: HTMLTableElement) {
     .getElementsByTagName("tbody")[0]
     .insertRow();
 
-  row.insertCell(-1).outerHTML = "<th>" + metal.symbol + "</th>";
+  const toggleButton = <HTMLInputElement>document.createElement("input");
+  const label = <HTMLLabelElement>document.createElement("label");
+  toggleButton.type = "checkbox";
+  toggleButton.classList.add("toggle");
+  toggleButton.id = "toggle_" + metal.idSuffix;
+  label.htmlFor = "toggle_" + metal.idSuffix;
+  const metalCell = <HTMLTableCellElement>document.createElement("th");
+
+  toggleButton.addEventListener("change", function () {
+    if (this.checked) {
+      (
+        document.getElementById(
+          "affinity_" + metal.idSuffix
+        ) as HTMLInputElement
+      ).disabled = true;
+      (
+        document.getElementById("bmc_" + metal.idSuffix) as HTMLInputElement
+      ).disabled = true;
+      metal.disableMetal();
+    } else {
+      (
+        document.getElementById(
+          "affinity_" + metal.idSuffix
+        ) as HTMLInputElement
+      ).disabled = false;
+      (
+        document.getElementById("bmc_" + metal.idSuffix) as HTMLInputElement
+      ).disabled = false;
+      metal.resetValues();
+    }
+    updateRow(metal);
+  });
+
+  metalCell.appendChild(toggleButton);
+  metalCell.appendChild(label);
+  const metalID = <HTMLParagraphElement>document.createElement("th");
```

Comment:
This shouldn't be a th element inside a th - can you change it to a span? (scss will also need to be changed.)

Reply:
nothing visually changed when I updated it to span so I didn't change the sass as the element the `metalID` spans to is still a `th`